packages/protobuf-bench/README.md

```diff
@@ -11,4 +11,4 @@ server would usually do.
 | code generator      | bundle size             | minified               | compressed         |
 |---------------------|------------------------:|-----------------------:|-------------------:|
 | protobuf-es         | 74,055 b      | 36,960 b | 9,651 b |
-| protobuf-javascript | 370,857 b  | 271,536 b | 43,759 b |
+| protobuf-javascript | 389,691 b  | 288,639 b | 45,153 b |
```

Comment:
We aren't updating google-protobuf anymore, right? Where does the increase in bundle size come from?

Reply:
`make bench` should fix it.

Reply:
Oddly no. It seems running `make` after removing those lock files changed them.

Reply:
Is it possible for `es-build` to affect it?

Reply:
Yes (you can look at the changelog), but very unlikely. 

It is much more likely that this is still using the newer version of google-protobuf. There we multiple lock files at one point, and now we probably have the wrong version in the remaining lock file. Note that the lock file is the authority in the installed version, the package.json file just has version constraints.

Reply:
Ah yes, you are right. The lock file was using a newer version since the version constraint was set at `^3.20.1`, so it generated the lock file with the latest version `3.21.2`. In this case, would it make sense to pin its version to `3.21.0` to match what it used to be, and generate a new lock file?